Model/IO/Export/Nastran/NastranExporter/NastranExporter.h

```diff
@@ -0,0 +1,93 @@
+//*************************
+//Created by Dominik Kopaczka
+//*************************
+
+#ifndef CPP_NASTRANEXPORTER_H
+#define CPP_NASTRANEXPORTER_H
+
+#include "../../MeshExporter.h"
+#include "../BulkData/ShortBulkData.h"
+#include"../BulkData/FreeBulkData.h"
+#include"../BulkData/LongBulkData.h"
+#include<iostream>
+#include<fstream>
+#include <iterator>
+#include<vector>
+#include<boost/filesystem/path.hpp>
+
+using std::string;
+using namespace MeshIS::Model::Export::Nastran::BData;
+namespace MeshIS
+{
+	namespace Model
+	{
+		namespace Export
+		{
+			namespace Nastran
+			{
+				class NastranExporter : public MeshIS::Model::Export::MeshExporter {
```

Comment:
Just comment: the class is ok, but source code looks quite ugly to me. It could be much simpler and clearer.